examples/DynamicCantileverCase/dynamic_cantilever_helper.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+class DynamicCantileverVibration:
+    def __init__(
+        self,
+        base_length,
+        base_area,
+        moment_of_inertia,
+        youngs_modulus,
+        density,
+        mode,
+        end_velocity_z=0.0,
+    ):
+        self.base_length = base_length
+        self.end_velocity_z = end_velocity_z
+
+        assert isinstance(mode, int)
+        assert mode >= 0
+        assert mode < 5
+        self.mode = mode
+
+        betas = np.array([0.596864, 1.49418, 2.50025, 3.49999]) * np.pi / base_length
+        self.beta = betas[mode]
+        self.omega = (self.beta ** 2) * np.sqrt(
+            youngs_modulus
+            * moment_of_inertia
+            / (density * base_area * base_length ** 4)
+        )
+
+        np_mode_l = self._compute_nonparametrized_mode(
+            base_length, base_length, self.beta
+        )
+        self.param = end_velocity_z / (-1j * self.omega * np_mode_l)
```

Comment:
Same for the `param` it is not clear which parameter is this?

Reply:
[3e1ca62](https://github.com/GazzolaLab/PyElastica/pull/173/commits/3e1ca62f44482de6dc719256fd47975ea3aa5037)